certified-connectors/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2149 @@
+                      "description": "internalId"
+                    }
+                  },
+                  "description": "result"
+                },
+                "face": {
+                  "type": "object",
+                  "properties": {
+                    "isIdentical": {
+                      "type": "boolean",
+                      "description": "isIdentical"
+                    },
+                    "confidence": {
+                      "type": "string",
+                      "description": "confidence"
+                    }
+                  },
+                  "description": "face"
+                },
+                "verification": {
+                  "type": "object",
+                  "properties": {
+                    "passed": {
+                      "type": "boolean",
+                      "description": "passed"
+                    },
+                    "result": {
+                      "type": "object",
+                      "properties": {
+                        "face": {
+                          "type": "boolean",
+                          "description": "face"
+                        },
+                        "notexpired": {
+                          "type": "boolean",
+                          "description": "notexpired"
+                        }
+                      },
+                      "description": "result"
+                    }
+                  },
+                  "description": "verification"
+                },
+                "authentication": {
+                  "type": "object",
+                  "properties": {
+                    "score": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "score"
+                    }
+                  },
+                  "description": "authentication"
+                },
+                "vaultid": {
+                  "type": "string",
+                  "description": "vaultid"
+                },
+                "matchrate": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "matchrate"
+                },
+                "executionTime": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "executionTime"
+                },
+                "responseID": {
+                  "type": "string",
+                  "description": "responseID"
+                },
+                "quota": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "quota"
+                },
+                "credit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "credit"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "default",

```

Comment:
@ccyy915 All the response description values of an operation must not be same. Please provide valid resposne descriptions